parmed/formats/sdf.py

```diff
@@ -27,10 +27,11 @@ def id_format(filename):
         is_fmt : bool
             True if it is a sdf file, False otherwise
         """
-        line = linecache.getline(filename, lineno=4)
         try:
-            return line.split()[-1] in ['V2000', 'V3000']
+            words = linecache.getline(filename, lineno=4).split()
+            return len(words) >= 3 and words[-1] in ('V2000', 'V3000')
         except IndexError:
+            # 4-th line is empty
             return False
```

Comment:
This doesn't have to be in a try-except block. If the 4th line is empty, `len(words) >= 3 and words[-1] in ('V2000', 'V3000')` will short-circuit and return `False`.  You literally *just* need those two lines :).